### ISSUE TITLE
Tooltip link option doesn't show text

### ISSUE THREAD

**Issue by AndrewBMartin** [michelson/dante2#247]
Dante2 version: 0.5.0-rc25

Browsers: Desktop Chrome and Mobile Safari

When highlighting text in the editor and selecting the link option. The background of the input field is white and the text color is also set to white so that you can't see what you're typing.

In the attached screenshot. I've typed a url, but you can't see it.

![Screen Shot 2020-11-21 at 3 15 50 PM](https://user-images.githubusercontent.com/5890950/99886761-0bda8780-2c0d-11eb-86d6-1d926dcc7d8e.png)

**Comment by michelson** [comment]
good catch, the html element that's that is overriding the bg color is a input with css class `.dante-menu-input` you could add `background: transparent` to it in order to fix this. 
If you pack that in a PR I will release it asap.

thanks for report this Andrew
